tools/mock-handling/src/index.ts

```diff
@@ -29,7 +29,7 @@ program
     .version(LIB_VERSION)
     .option("--recordingDir <path>", "Directory with recording inputs and outputs")
     .option("--project-id <id>", "Project id from which you want to capture mock data")
-    .option("--username <email>", "Your username that you use to log in to GoodData platform.")
+    .option("--tigerToken <token>", "GoodData tiger platform auth token.")
     .option("--hostname <url>", `Instance of GoodData platform. The default is ${DEFAULT_HOSTNAME}`)
     .option("--config <path>", `Custom config file (default ${DEFAULT_CONFIG_FILE_NAME})`)
     .option("--backend <type>", `Backend (default ${DEFAULT_BACKEND})`)
```

Comment:
I expect backend option should be removed?

Reply:
in internal tooling is only allowed tiger